### PR TITLE
feat: support setting story type on creation

### PR DESCRIPTION
**Pull request by alexdao3**

### CODE REVIEW

File: org-clubhouse.el [hunk 1]
```diff
@@ -404,6 +421,35 @@ If unset all projects will be synchronized")
...
+   "Select a default story type: "
+   (-map #'cdr org-clubhouse-default-story-types)
+   :history 'org-clubhouse-default-story-history
+   :action (lambda (selected)
+             (let ((story-type
+                    (->> org-clubhouse-default-story-types
+                         (-find (lambda (proj)
+                                    (string-equal (cdr proj) selected)))
+                         car)))
+                         (if (string-equal story-type "prompt")
```
Comment (glittershark) [inline org-clubhouse.el]: this indentation looks a little off

File: org-clubhouse.el [hunk 2]
```diff
@@ -404,6 +421,35 @@ If unset all projects will be synchronized")
...
+  (ivy-read
+   "Select a default story type: "
+   (-map #'cdr org-clubhouse-default-story-types)
+   :history 'org-clubhouse-default-story-history
+   :action (lambda (selected)
+             (let ((story-type
+                    (->> org-clubhouse-default-story-types
+                         (-find (lambda (proj)
+                                    (string-equal (cdr proj) selected)))
+                         car)))
```
Comment (glittershark) [inline org-clubhouse.el]: :bulb: looks like there's a common helper function waiting to be extracted here... no worries if you don| feel like it, though.

File: org-clubhouse.el [hunk 3]
```diff
@@ -404,6 +421,35 @@ If unset all projects will be synchronized")
...
+(defun org-clubhouse-prompt-for-default-story-type ()
+  (interactive)
+  (ivy-read
+   "Select a default story type: "
+   (-map #'cdr org-clubhouse-default-story-types)
+   :history 'org-clubhouse-default-story-history
+   :action (lambda (selected)
+             (let ((story-type
+                    (->> org-clubhouse-default-story-types
+                         (-find (lambda (proj)
```
Comment (glittershark) [inline org-clubhouse.el]: this local variable should probably indicate that its a story type.

**Comment by alexdao3** [comment]
PR comments addressed. Hopefully it's ready for merge!

**Pull request merged**